poincare/include/poincare/expression.h

```diff
@@ -286,6 +285,9 @@ class Expression : public TreeHandle {
 
   static void Tidy() { sSymbolReplacementsCountLock = false; }
 
+  /* Tuple */
+  typedef std::initializer_list<Expression> Tuple;
```

Comment:
I find it clearer with `std::initializer_list<Expression>` or `std::initializer_list<TreeHandle>` instead of Tuple

Reply:
I don't 😄